dev-scheme/sigscheme/sigscheme-0.8.5-r1.ebuild

```diff
@@ -10,8 +10,14 @@ SRC_URI="https://${PN}.googlecode.com/files/${P}.tar.bz2"
 LICENSE="BSD"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
+
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
 
 src_configure() {
-	econf --enable-hygienic-macro
+	econf --enable-hygienic-macro -disable-static
```

Comment:
is this correct (single hypen)?

Reply:
I meant to insert two dashes. It accepts even one dash, that's why I didn't notice it at first, but changed it to double for consistency